commands/channel.js

```diff
@@ -613,7 +616,63 @@ module.exports = {
 						await interaction.message.edit({ components: buttons });
 					});
 			}
+			case 'set_jtc_status': {
+				const modal = new ModalBuilder()
+					.setCustomId(global.getButtonIdString('channel', 'set_jtc_status', [interaction.channel.id]))
+					.setTitle('Voice Channel Status');
+				const statusInput = new TextInputBuilder()
+					.setCustomId('status')
+					.setLabel('Enter the new voice channel status:')
+					.setMaxLength(30)
```

Comment:
Any reasonable reference for this? The API we're using is already undocumented (even in the official discord docs) so I don't actually know what the limit is.

Reply:
I tried a few different combinations for the status message, and the maximum length was about 30 characters before the message was cut off.

Reply:
"about" :D good enough

Reply:
The default was 400(?), so this was more sane. The line breaks based on total character width rather than the number of characters.